### ISSUE TITLE
borg stickers?

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#1902]
Green on black (similar to logo on docs site), for laptops. Is that interesting?

If you'ld want one "heart" this post.

**Comment by ThomasWaldmann** [comment]
https://www.stickermule.com/custom-stickers

**Comment by ThomasWaldmann** [comment]
If someone wants to help: we need a data file to upload to wir-machen-druck.de, see there for the requirements (german):

https://www.wir-machen-druck.de/faq-qualitaetskontrolle.html

Maybe these green neon stickers would be a nice base material?

Logo same font / close colour like there: https://borgbackup.readthedocs.io/en/stable/

Text:
- Borg (Logo) + "www.borgbackup.org" below it
- optionally "You will be assimilated!"?

If someone has time to work on this, help would be appreciate!
But please tell before you start to avoid double work.

**Comment by klawdhfzasjhaa** [comment]
Hi there,

any new info about the stickers? I'm a newbie borg user, but very happy with it! :)
I just downloaded the logo.svg, which will have the perfect quality for this purpose. Could you please tell me the font, so I can create the subtext accordingly?

It would be great to simply design the sticker and let everyone order them individually. In my first quick search I was not able to find a service who offers this. Another approach could be: provide the ready-to-print-data here or better on the website for everyone to use.

**Comment by ThomasWaldmann** [comment]
No news.

Font: would have to start searching for it (issue tracker, mailing list, irc logs).

Rectangular sounds good. It's a borg cube, so round seems strange. :-)

About individual orders: guess that is rather a good idea for more expensive stuff, like hoodies/shirts. Stickers are relatively cheap and manufacturers / dealers usually don't want to deal with small orders, like for a few stickers. So I think the way to go is to use some organization funds and produce a medium amount of stickers and spread them at conferences and hacker spaces.
If anyone could order such amounts, that still would be good for wide / international distribution, though.

**Comment by klawdhfzasjhaa** [comment]
Found the font [here](https://github.com/borgbackup/borg/blob/master/docs/_static/logo_font.txt): It's [Black Ops One](https://www.1001fonts.com/black-ops-one-font.html). The color is: #00DD00 (0,221,0).

True: Borg = cube ~ reactangle. ;)
Here's my first try:
![overview_a](https://user-images.githubusercontent.com/2808094/53126504-9685e280-3560-11e9-8a09-0c41c79b06e3.png)
What do you and all the others think? Maybe we got someone with decent graphics skills here. :)

**Comment by ThomasWaldmann** [comment]
They somehow look too narrow, like horizontally condensed, compared to there: https://www.borgbackup.org/

When thinking about the goal of such stickers (make borg more known, I mean borgbackup, not the guys from startrek) and assuming that the main target audience is people **not** knowing borgbackup yet, I think we need to include www.borgbackup.org somehow or else it will be seen as startrek borg advertisement.

**Comment by klawdhfzasjhaa** [comment]
> horizontally condensed: 
True, I didn't notice that. I used the logo.svg first instead of creating "Borg" from the font.

>  www.borgbackup.org:
Added. Now I think it's too much. Maybe we should only use either "www.." or "You will .." as you can see in the second sample:

![borglogo_assi_BlackOpsOneB_Black_02](https://user-images.githubusercontent.com/2808094/54455568-75c63c80-475c-11e9-9613-c6e469945195.png)
![borglogo_www_BlackOpsOneB_Black_01](https://user-images.githubusercontent.com/2808094/54455569-75c63c80-475c-11e9-983f-897ac83f76e0.png)

**Comment by ThomasWaldmann** [comment]
Yeah, agreed, the upper one is a bit much.

Can you try using the lower one and the putting the "You will be assimilated!" in a small font (size as the "www...." part?) as a single line below everything?

**Comment by klawdhfzasjhaa** [comment]
I don't like it that much:
![borglogo_www_assi_BlackOpsOneB_Black_01](https://user-images.githubusercontent.com/2808094/54460352-9a74e100-4769-11e9-8a9d-11268493a3e4.png)
No. 2 of the post above is my fav at the moment.

But maybe we should ask a real designer. ;)

**Comment by ThomasWaldmann** [comment]
Not sure if we have any "real designer" here (if so, tell us!). But we can ask the community for feedback...

**Comment by ThomasWaldmann** [comment]
Heart this if you like A.

**Comment by ThomasWaldmann** [comment]
Heart this if you like B.

**Comment by ThomasWaldmann** [comment]
Heart this if you like C.

**Comment by Perflyst** [comment]
C.. because if you do not want the text below the logo you can just cut it off yourself. If you cut it off you have basically B

**Comment by ThomasWaldmann** [comment]
@klawdhfzasjhaa can you attach the original documents (svg?) for B) and C), please?

We also need a good / easy manufacturer for these stickers, does anybody have recommendations?

Coolest would be if we could upload a design and people could order them internationally in small to medium quantities for cheap. :-)

Soon is GPN 2019 ("Gulasch Programmier Nacht"), could distribute some there. :-)

**Comment by frommMoritz** [comment]
> We also need a good / easy manufacturer for these stickers, does anybody have recommendations?

Someone gave me stickers recently, ordered on [Flyeralarm](https://www.flyeralarm.com/de/shop/configurator/index/id/34/outdoor-aufkleber.html).Tthose seem to have a decent quality. Depending on your budget you can get a good a mount of stickers for your money.

> Soon is GPN 2019 ("Gulasch Programmier Nacht") and EuroPython 2019, could distribute some there. :-)

I'm not at those events but would be happy with some stickers. Would it be possible to "just" send you a stamped return envelope in order to get some?

**Comment by mlbarrow** [comment]
I have used StickerMule (mentioned earlier in this ticket) here in the US and love their stuff. The Perl Weekly newsletter has a discount through StickerYou, so there's another source.

**Comment by klawdhfzasjhaa** [comment]
Hey there. I've created them with GIMP, which does not natively support SVG export. You can easily transform it to JPG, EPS and so on if needed. Also, if the shop has black stickers, we can delete the black background from the files. When we think of black laptop lids it makes sense to have a transparent sticker foil with the green letters on it. 

[stickers.zip](https://github.com/borgbackup/borg/files/3010009/stickers.zip)

> Coolest would be if we could upload a design and people could order them internationally in small to medium quantities for cheap. :-)

That was my intention with this:

> It would be great to simply design the sticker and let everyone order them individually.

**Comment by ThomasWaldmann** [comment]
flyeralarm.com:
-  product: https://www.flyeralarm.com/uk/shop/configurator/index/id/5803/removable-stickers.html
- format: A8 or A7? (landscape)
- requirements: see data sheet there on the product page
- more requirements: https://www.flyeralarm.com/uk/content/index/open/id/1569/artwork-requirements.html
- looks like ordering some thousand stickers is doable
- but we should be pretty sure about the uploaded file before producing such an amount and I must admit that I do not fully understand all these requirements / their relevance to us.

**Comment by ThomasWaldmann** [comment]
stickermule.com:
- seems quite a bit more expensive, so we would be limited to a few hundreds
- but users could get e.g. 10 stickers for 20 bucks and intl. shipping
- free help with artwork
- 3" x 2"
- wants vector or 300 dpi raster
- free intl shipping (to some countries depending on order amount)

**Comment by ThomasWaldmann** [comment]
wir-machen-druck.de:
- german-only website?
- https://www.wir-machen-druck.de/neonaufkleber-10-einfarbig-schwarz-bedruckt-mit-freier-groesse-rechteckig.html
- if that neon-green material should be the sticker text, we would need to print black everywhere except where it should be green? guess we can't do that with a vector / font, just with raster?

**Comment by ThomasWaldmann** [comment]
If someone has a clue about print, help me decide about the service / product.

**Comment by klawdhfzasjhaa** [comment]
Maybe someone can take a USB drive with the RAW data and go to a local store to clarify / check the requirements. I guess the output is better this way than having a bunch of rookies (us ;) ) trying to create a good sticker online by ourselves. Does someone here live in a bigger city like Berlin oder Stuttgart?

**Comment by marcpope** [comment]
I have 20+ years of print, pre-press an design experience. Glad to help. If you give me the place you want them made, I will give you the correct file format.

**Comment by ThomasWaldmann** [comment]
@marcpope thanks a lot for offering help!

I just had a look again and I think we go for `flyeralarm.com`, see https://github.com/borgbackup/borg/issues/1902#issuecomment-476864174.

Also, I checked with existing stickers on my laptop: A8 is too small, so guess we should go for A7.

Image: The "C" version. I like the "if you don't like the assimilation phrase, you can cut it off" approach - maybe there should be a few mm more distance between logo and that phrase to ease that?

**Comment by ThomasWaldmann** [comment]
@frommMoritz sure, sending a stamped return envelope would work.

**Comment by ThomasWaldmann** [comment]
5 weeks left until GPN 2019, see #4510 - can we manage to have stickers ready for that?

**Comment by ThomasWaldmann** [comment]
related: #4527 - if somebody wants more stickers to be spread, increase the bounty there. :-)

**Comment by ThomasWaldmann** [comment]
@marcpope only little time left until GPN 2019 conference, did you already start working on the files (see above)?

**Comment by ThomasWaldmann** [comment]
ping @marcpope

**Comment by dragetd** [comment]
wir-machen-druck has rather poor quality. Yes, they are cheap and the printing quality itself is OK, but not the stickers. The colors flake off and the stickers look really poor. :(

I'd happily add some funds for the merge if I get a few hundred stickers to spread (e.g. on the camp - just got my ticket \o/)

**Comment by ThomasWaldmann** [comment]
@dragetd thanks for the info.

@marcpope are you still alive?

**Comment by ThomasWaldmann** [comment]
OK, guess @marcpope does not have time to spend currently.

Who else can make the files needed?

**Comment by bebehei** [comment]
As my dad's profession is in the publishing industry, I know some graphic designers. I'm regularly servicing the system from one of those. I may ask her for support. She's working right in center of Stuttgart also she has got plenty of experience with flyeralarm.

AFAIK, flyeralarm has got some weird document requirements, but I guess she's familiar with it.

I haven't read the full thread yet. Which documents shall I use to approach her?

**Comment by marcpope** [comment]
For some reason I wasn't getting notifications. I can definitely work on it.  Does anyone know what font the borg type is? The files that were attached here I can't open in any Adobe Products.

**Comment by ThomasWaldmann** [comment]
@marcpope font see there: https://github.com/borgbackup/borg/issues/1902#issuecomment-465766222

@bebehei thanks for the offer. guess it's enough if 1 person is working on it, so let's give @marcpope a chance.

https://github.com/borgbackup/borg/issues/1902#issuecomment-473451280 see that comment and the design C above.

**Comment by marcpope** [comment]
![image](https://user-images.githubusercontent.com/11093841/59162617-fbd6f280-8ac1-11e9-8d6e-60015c874806.png)
So we are going with this design right?

**Comment by ThomasWaldmann** [comment]
Basically yes.

Maybe check if there is enough space to cut away the "You will be assimilated" and still have enough lower border left (and if not, leave a bit more space in between that phrase and the stuff above it).

**Comment by klawdhfzasjhaa** [comment]
@marcpope 
Don't know, if it's too late..But attached you'll find eps and psd files for whatever fancy Adobe program you use. ;)

[stickers.zip](https://github.com/borgbackup/borg/files/3273297/stickers.zip)

**Comment by marcpope** [comment]
Thanks Neo.

That makes it easier. One thing to note. The bright green color won’t be bright green in these stickers. They only use CMYK. to make green, we mix Yellow and Cyan together which gives a much duller green. The right way to go about it would be to use a 2-color print sticker where we could pick a nice bright green. It does not seem like this company does special color stickers though. Please let me know if the CMYK (100yellow and 100cyan) is ok or should we shoot for a different printer?

-Marc

**Comment by ThomasWaldmann** [comment]
@marcpope interesting hint (good to have somebody who is experienced with such stuff, thanks!).

I'ld like to stay with flyeralarm (good price and decent quality AFAIK), so guess we need to match some borg-ish green as good as possible with CMYK.

As I had no clue, I just searched a bit on the web and found that:

https://www.quora.com/How-do-I-get-bright-colors-in-CMYK

There in the comments is the hint that:
```
Whether you get bright colors or dark colors simply depends on which
colors you mix. CMYK 61, 0, 100, 0 for example would be a bright green.
```

The author of the original post rather tends to fully saturated colours (100), but mixing a fully saturated yellow with a bit less saturated cyan seems to make sense to me.

But, I don't have any experience with that nor do I have a calibrated monitor to check it.

**Comment by marcpope** [comment]
That’s probably decent. I will use a 4 color black so that it’s as much contrast as possible.

**Comment by elho** [comment]
Just stumbled across this issue...

> flyeralarm.com:
> stickermule.com:

At work, we have had various stickers (e.g. [Elephant Shed](https://twitter.com/credativDE/status/1006808432807161856) and [pg_snakeoil](https://twitter.com/credativDE/status/1111191220867526656) ones) made and tested both of these print services, the stickermule ones, while indeed more expensive, are higher quality material, so we did stick (pun not intented, at least at first... :stuck_out_tongue_winking_eye:) with those.

**Comment by ThomasWaldmann** [comment]
@marcpope any news?

**Comment by Tobaloidee** [comment]
hey @ultrabug  will try to work on this and post here when done. :)

**Comment by ThomasWaldmann** [comment]
@ultrabug @Tobaloidee please continue in the new ticket #4690, this ticket here is about making stickers.

**Comment by ThomasWaldmann** [comment]
@bebehei can you please work (potentially in parallel to @marcpope) and we'll just use the stuff from whoever finishes first?

Would be nice if we had some stickers before before the planet melts down. :-)

**Comment by marcpope** [comment]
I am unable, wife having complications from pregnancy and were at the hospital.. sorry!

**Comment by ThomasWaldmann** [comment]
@marcpope wish you three the best!

**Comment by bebehei** [comment]
Sorry, I just recently moved cities and I'm not really able to work with her for that stuff.

**Comment by ThomasWaldmann** [comment]
OK, seems like nobody wants / can do this, closing, no stickers.

**Comment by dragetd** [comment]
@ThomasWaldmann I understand your frustration about the turn of events. It happens often that people try to start something but that other things take priority and projects are not finished.

I suffer from this issue myself a lot, plus my media-design skills are only average. So I did not even try to get involved as I was afraid of suffering the same fate of not completing it. But there are defiantly people who would like Borg stickers.

I want to be really careful with promises (and rather hope for someone else picking up the idea again), but before this issue is closed and forgotten, I'd rather ask my graphics-friend-of-choice to help me and have a go myself.

I'd be willing to trust out a few hundred euros, take care of ordering and bringing them to the congress in Leipzig. I might fail too, due to reasons(tm), but it would be nice if we could give this a shot again at least for the congress.

**Comment by dragetd** [comment]
What do you think about a non-rectangular version? I madea mockup, the final version will have more precise pixel placements:

![shaped_transparent](https://user-images.githubusercontent.com/3639577/64283947-c1fa7880-cf58-11e9-8699-ca8efcfe17ec.png)

Or alternatively with the URL more noticable:
![shaped_transparent_white](https://user-images.githubusercontent.com/3639577/64284312-a348b180-cf59-11e9-8bd6-1bdfb74c8526.png)



Stickermule is certainly a bit more expensive, but the quality should be okay at least.
The width could be 5cm:
* 2000 pcs. 304€
* 3000 pcs. 405€
* 5000 pcs. 590€

or 7,5cm:
* 2000 pcs. 512€
* 3000 pcs. 696€
* 5000 pcs. 1033€

I'd be willing to keep up to 400 myself (100 pcs. at least). If people I trust commit to taking/paying the rest (e.g. taking 1900 stickers when we get 2k) at the congress (payment at the congress is fine), I'd be willing to trust out up to the 590€. :-)

If Thomas says he wants more or some money is collected in advance, we could even increase that.

What do you think?

**Comment by ThomasWaldmann** [comment]
Non-round looks a bit more pretty / advanced, but it is also more hard / expensive to cut and to align with your laptop cover / existing sticker on there I guess.

People also liked the option of keeping-or-cutting-away the "you will be assimilated" in the original design, which is easier with a rectangular design.

I think it would be good to collect some funds in advance, there is a separate ticket already with a bounty on it, so that bounty can be just increased. Thanks to everybody contributing!

Quality: I guess I would prefer good stickers with a more plastic-y touch than stickers being more like plain cheap paper. Also I dislike sticker leaving a lot of hard-to-remove glue and paper when removed.

I have no idea about the quality offered by different companies / products though (and one also maybe should not rely on "expensive equals good quality").

**Comment by dragetd** [comment]
Flyeralarm 'good quality' paper is '90 g Strukturpapier perlmutt', the highest option I can take (the others are 50/60/72 g). I have no comparison and unless someone else who knows tells me what to order, I would go for this option.

Personally I also favor the pre-cut variant I posted - I could/will still make the corners with a smaller radius so they look more 'sharp'. I like free-form sticks a lot more, but this is a personal thing.

But if you say you want the rectangular ones with the text (you will be assimilated), then I will take the file as it is posted up there (ask my buddy for help with the color conversions) and print that one!

**Comment by dragetd** [comment]
Rectangular stickers are also almost only half the price (compared to what I said above).

@ThomasWaldmann It is your project and I want you to choose if you want the rectangular or custom-shaped ones. Or I'll just order 3k of each (which will be <300€ total).

I need your final word. :)

**Comment by ThomasWaldmann** [comment]
Let's go for the cheaper rectangular option (with the easy "you will be assimilated" cut-away option).

Maybe adjust the borders so the text/logo size and border size are appropriate, considering the cut away after which it should also look ok.

Please try to get good material (I have no experience with the misc. services, but I like stickers which can be removed without them disintegrating and leaving a glue/paper mess behind on the laptop).

Thanks for dealing with this.

BTW, I'll be away for some days - just decide what's best if there is anything else.

**Comment by dragetd** [comment]
Yes, I got one of the sample-booklets from Flyeralarm, I will pick one of the better materials. Will do. :)

**Comment by ramses0** [comment]
A comment w.r.t. spacing (as you're getting to pixel-perfect), there are a few rough bits which I've highlighted in the following image:

<img width="906" alt="Screen Shot 2019-10-31 at 12 27 00 PM" src="https://user-images.githubusercontent.com/110406/67979650-f47fc500-fbd9-11e9-8930-0f9177554a2d.png">

1) likely the "x-height" of the URL should be very similar to the lower stroke-width of the "G"

2) the edge of the B and corner of the G seem like other useful guideposts for aligning the URL

3) as an option for including the "you will be assimilated" text/motto, there's a convenient spot between the "O" and "G" corners that fits the text where it's readable but not overpowering either the URL or detracting from the main project name, and keeps the sticker very square.

I grabbed one of the  other logos and roughed it out via pixels in inkscape (definitely not print-ready), but whoever ends up doing the print run may be interested in the above suggestions.

**Comment by ThomasWaldmann** [comment]
PSA: there are only 8 weeks left until 36c3 (big CCC congress, ~17.000 visitors).

This would be a great opportunity to internationally spread some sticker **if** we get them finally done **now**.

**Comment by dragetd** [comment]
I was under the assumption that we spread them at the congress. I have some more orders lined up (also for other things at the congress), thus did not rush things. Sorry Thomas, was not aware that you wanted them urgently before the congress! The order is prepared but not sent out yet. I will send it out ASAP.

**Comment by ThomasWaldmann** [comment]
I don't need them before congress. I just had the impression that nothing is happening and we might fail again having them there.

So, sorry for disturbing and thanks for caring about that!

**Comment by dragetd** [comment]
I got 5k and will bring all of them. Then people can decide how many they 'buy' off me (were 2-3 cents each). :-)

**Comment by szpak** [comment]
@dragetd Do you - by any chance - plan to be also at [FOSDEM](https://fosdem.org/2020/) this year?

**Comment by ThomasWaldmann** [comment]
@dragetd looks like you could close #4527 and collect the bounty?

**Comment by dragetd** [comment]
> @dragetd Do you - by any chance - plan to be also at [FOSDEM](https://fosdem.org/2020/) this year?

Nope, sorry.

For those who are not on the congress: Maybe find someone you know who can fetch a bunch for you there. :)

@TW Let us sort it out on the congress. Sorry for the whole late replies and stuff. Looking forward to the congress. :)

**Comment by ThomasWaldmann** [comment]
@dragetd ok, see you there! :-)

**Comment by dragetd** [comment]
TW has the majority of the stickers now and will take them to events.

I have kept a smaller bunch myself. If you are some hackspace or are one of the bounty source donators or similar, you can poke me privately and I will mail some! Sorry, not for individuals.

**Comment by dragetd** [comment]
This issue can be closed. :) The offer from above still stands, tho. Drop a note and I will mail a bunch! :)

**Comment by szpak** [comment]
@dragetd / @ThomasWaldmann you might mention the conferences you will be attending the following years for people willing to meet you (and the stickers) there :).

**Comment by dragetd** [comment]
\*lets out a soft sigh\* Conferences… yeah. We will see if the chaos communication congress will take place this year, I am not 100% sure about this currently. If it will, this will be most likely the next event. But I got my hopes not very high. :(

Which hackspace is closest to you? Poke them and ask if I may send a bunch for free, then I will do that :D

**Comment by ThomasWaldmann** [comment]
Guess we should not hold our breath until conferences, hackerspace meetings and other public events / get-togethers happen again. So guess there is currently no good and efficient way to distribute stickers.

**Comment by ThomasWaldmann** [comment]
One TODO here is to commit the source and print-ready stuff for the stickers to the borg repo (IIRC we do not have that yet, right?). Or to another repo, in case it is rather big.

**Comment by dragetd** [comment]
Ah right, you told me that on the congress. Will send a PR soonish!
